frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -133,24 +133,32 @@ export const getHelmActionConfig = (
   switch (helmAction) {
     case HelmActionType.Install:
       return {
+        type: HelmActionType.Install,
         title: 'Install Helm Chart',
-        subTitle: 'The helm chart will be installed using the YAML shown in the editor below.',
+        subTitle: {
+          form:
+            'The Helm chart can be installed by completing the form. Default values may be provided by the Helm chart authors. ',
+          yaml:
+            'The Helm chart can be installed by manually entering YAML or JSON definitions, or by dragging and dropping a file into the editor. ',
+        },
         helmReleaseApi: `/api/helm/chart?url=${chartURL}`,
         fetch: coFetchJSON.post,
         redirectURL: getOriginRedirectURL(HelmActionOrigins.topology, namespace, releaseName),
       };
     case HelmActionType.Upgrade:
       return {
+        type: HelmActionType.Upgrade,
         title: 'Upgrade Helm Release',
         subTitle:
-          'Upgrade by selecting a new chart version or manually changing the YAML shown in the editor below.',
+          'Upgrade by selecting a new chart version or manually changing the YAML shown in the editor below. ',
```

Comment:
Why do 3 of these strings have a trailing space?
If spacing is needed it should be up to the consumer of the string to add space between elements.

Reply:
Updated.